contact.html

```diff
@@ -0,0 +1,164 @@
+              <label for="email" class="form-label">Adresse courriel :</label>
+              <input type="email" class="form-control text-center" id="email" name="email" required>
+            </div>
+            <div class="mb-3">
+              <label for="email" class="form-label">Numéro de téléphone :</label>
+              <input type="tel" class="form-control text-center" id="telephone" name="telephone" required>
+            </div>
+            <div class="mb-3 form-group">
+              <label for="subject" class="mb-3">Sujet:</label>
+              <select id="subject" name="subject" class="form-control text-center" required>
+                <option value="" disabled selected>Choisissez une option</option>
+                <option value="support">Support technique</option>
+                <option value="inquiry">Demande de renseignements</option>
+                <option value="collaboration">Proposition de collaboration</option>
+                <option value="billing">Problème de facturation</option>
+                <option value="quote">Demande de devis</option>
+                <option value="improvement">Suggestion d'amélioration</option>
+                <option value="other">Autre</option>
+              </select>
+            </div>
+            <div class="mb-3">
+              <label for="message" class="form-label">Message :</label>
+              <textarea id="message" class="form-control text-center" name="message" required></textarea>
+            </div>
+            <div class="text-center pt-3">
+              <button class="btn btn-light border-4 border-light" type="submit">Soumettre</button>
+            </div>
+          </form>
+
+          <div class="go-to-maps">
+            <h4>Où me trouver!</h4>
+            <img src="/assets/icons8-up-arrow-100.png">
+          </div>
+
+        </div>
+      </div>
+
+      <div class="card bg-dark text-light border-5">
+        <div id="map"></div>
+        <div class="p-3">
+          <strong>Cégep Garneau</strong>
+          <address>Bd de l'Entente, Québec City,<br />Quebec G1S 4S3</address>
+        </div>
+
+      </div>
+    </div>
+  </main>
+
+
+
+  <footer class="border-4 border-top">
+    <div class=" pt-5 text-center bg-light">
+      <div class="card-body align-items-end">
+        <div class="container">
+          <div class="row justify-content-center">
+            <div class="col-md-4">
+              <h4>Compagnie</h4>
+              <ul class="list-unstyled">
+                <li><a href="#" class="text-dark">Coordonnées</a></li>
+                <li><a href="#" class="text-dark">Confidentialité</a></li>
+                <li><a href="#" class="text-dark">Utilisation</a></li>
+              </ul>
+            </div>
+            <div class="col-md-4">
+              <h4>Me joindre</h4>
+              <ul class="list-unstyled">
+                <li><a href="https://www.facebook.com/raphael.doucet.39/" class="text-dark">Facebook</a></li>
```

Comment:
For external links, try to user `target='_blank'` property to open them on a new tab.